### DISCUSSION TITLE
How can I add openAI api key to run examples?

### DISCUSSION THREAD

**Discussion by henomis** [henomis/lingoose#83] [Q&A]
LinGoose uses env variable `OPENAI_API_KEY` by default. So you need to add to your shell rc file (eg. .zshrc, .bashrc) the following line:

```sh
export OPENAI_API_KEY=<your openai key>
```
 > Hope this helps! 🪿

**Comment by Darwinium** [comment]
Oh my, I spent 15 minutes trying to figure out the correct name of the variable. Highly recommend adding this point right in the "Quick start" section. Anyway, thanks for your work. Still trying to understand if it makes sense to use or go with the langchain but the first impression is good.

**Reply by henomis** [reply]
Thank you for your feedback! I'll add as you suggested.
